common/schemas.py

```diff
@@ -1,4 +1,17 @@
+from dataclasses import dataclass
+
 # Numbers represent the prefix of the id of the cards.
 # i.e. 26000037 is a Inferno Dragon, 27000007 is Elixir Collector, 28000000 is fireball
 CARD_TYPE_ID_PREFIX = {'26': 'troop', '27': 'building', '28': 'spell'}
 MAX_CARD_LEVEL = 14
+
+# Data Clases
```

Comment:
Is this comment necessary? hahahaha The annotation already indicates that it's a dataclass lmaooo